### ISSUE TITLE
can i  add nesbox to my blog?

### ISSUE THREAD

**Issue by bazaar24** [nesbox/emulator#39]
can i  add nesbox to my blog? (tumblr)

**Comment by nesbox** [comment]
If you have access to html code then yes

**Comment by bazaar24** [comment]
I have access to html, but I can only upload files to an external host as dropbox, mediafire... it's possible?

**Comment by colt05** [comment]
yes as long as you can provide a direct link to the files you uploaded
